cfn/README-GD.md

```diff
@@ -55,7 +59,7 @@ An example of a successful response is:
 }
 ```
 
-**Important:** If the command returns no output, verify your Alert Logic account has administrator permissions. Click [here](https://console.cloudinsight.alertlogic.com/api/aims/) for more information about AIMS APIs.
+**Important:** If the command returns error about not having necessary role, verify your Alert Logic account has administrator permissions. Click [here](https://console.cloudinsight.alertlogic.com/api/aims/) for more information about AIMS APIs.
```

Comment:
I've changed this one as it looks a bit outdated: now if role is wrong, there is message

```
The $AL_USERNAME doesn’t have Administrator role. Assigned role is '$ROLES
```

so "no output" sounds misleading to me.

Reply:
Maybe something like:
"If the command returns an error about not having the necessary role, verify..."
?

BTW, is the AIMS acronym known to the user?